### PR TITLE
Added social share tab

### PR DESCRIPTION
**Pull request by danthaman44**
-Added "Share" tab to merchant page with social icons
-Removed Social icons from OwnerPanel
-Added confirmation tooltip to "Copy to clipboard" icon

Test plan:
Screen Shot 2020-06-20 at 10 12 03 PM
Screen Shot 2020-06-20 at 10 23 06 PM
share-confirmation

### CODE REVIEW

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    font-size: 16px;
+    line-height: 22px;
+    align-items: center;
+    letter-spacing: 0.15em;
+    text-transform: uppercase;
+    color: #000000;
+    padding-left: 10px;
+`;
+
+const SocialContainer = styled.div`
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:137]: Could you add cursor: pointer here? Or does default already handle it

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    margin: 5px 0px 5px 0px;
+`;
+
+const ClipboardContainer = styled.div`
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:141]: Same here

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+            <SocialContainer>
+                <ClipboardContainer ref={target}>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <ClipboardIcon
+                            size={28}
+                            style={clipboardStyle}
+                        />
+                    </CopyToClipboard>
+                    <CopyToClipboard text={location} onCopy={() => setShow(!show)}>
+                        <SocialTextLabel>COPY TO CLIPBOARD</SocialTextLabel>
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:93]: Should we localize?

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                    <Overlay target={target.current} show={show} placement="bottom">
+                        {(props) => (
+                            <Tooltip id="overlay-example" {...props} style={{
+                                backgroundColor: socialIconForegroundColor,
+                                color: socialIconBackgroundColor,
+                                padding: '5px 15px 5px 15px',
+                                borderRadius: 3,
+                                ...props.style,
+                            }}>
+                                COPIED!
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:104]: Same we should localize here

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+            <SocialContainer>
+                <EmailShareButton url={location} className="share">
+                    <SocialIcon
+                        network="email"
+                        fgColor={socialIconForegroundColor}
+                        bgColor={socialIconBackgroundColor}
+                        style={socialIconDimensions}
+                    />
+                    <SocialTextLabel>EMAIL</SocialTextLabel>
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:80]: localize

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+export interface Props {
+    seller: BrowsePageSeller;
+}
+
+const StoreSocial: React.SFC<Props> = ({ seller }) => {
+    const [show, setShow] = useState(false);
+    const target = useRef(null);
+
+    const location = window.location.href;
+    const facebookQuote = 'Help raise money for ' + seller.name;
```
Comment (justintmckibben) [inline src/components/StoreInfo/StoreSocialTab.tsx:25]: Also here too

File: src/components/StoreInfo/StoreSocialTab.tsx [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+};
+
+const HeaderText = styled.div`
+    font-family: Open Sans;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 22px;
+    color: #1E1E1E;
+    margin: -10px 0px 30px 10px;
```
Comment (jacoblb64) [inline src/components/StoreInfo/StoreSocialTab.tsx:121]: what's with this negative margin?

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 1 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:137]: Done

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 3 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:93]: Yes definitely! Will put out a followup PR for that

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 7 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:121]: Its to reduce the spacing between the header text and the top of the container, the container puts too much space there by default
Screen Shot 2020-06-21 at 12 19 38 AM

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 2 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:141]: Done

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 7 above)
Comment (jacoblb64) [reply src/components/StoreInfo/StoreSocialTab.tsx:121]: Makes sense, I feel like negative margin is usually a sign of something needing fixing somewhere else, maybe we should reduce the upper container's size--do the other tabs have this problem? Anyway this is super minor so I wouldn't worry too much about it.

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 3 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:93]: https://trello.com/c/hw16UFsN/506-add-localization-to-merchant-tabs

File: src/components/StoreInfo/StoreSocialTab.tsx (hunk 7 above)
Comment (danthaman44) [reply src/components/StoreInfo/StoreSocialTab.tsx:121]: Totally agree, I'll followup with @ath-chen about reducing the container size.

Comment (jacoblb64) [reply src/components/StoreInfo/StoreSocialTab.tsx:121]: 👍 Sounds good man

**Pull request merged**